src/Umbraco.Web.UI.Client/src/loader.dev.js

```diff
@@ -10,7 +10,9 @@ LazyLoad.js(
         'lib/angular/1.1.5/angular-mobile.js',
         'lib/angular/1.1.5/angular-sanitize.min.js',
 
-        'lib/angular/angular-ui-sortable.js',
+        'lib/angular/ng-caps-lock.js',
```

Comment:
This is also required in /Umbraco.Web/UI/JavaScript/JsInitialize.js for c# server side.